homeassistant/components/cloud/prefs.py

```diff
@@ -63,17 +64,19 @@ async def _async_migrate_func(
 class CloudPreferences:
     """Handle cloud preferences."""
 
-    def __init__(self, hass):
+    def __init__(self, hass: HomeAssistant) -> None:
         """Initialize cloud prefs."""
         self._hass = hass
         self._store = CloudPreferencesStore(
             hass, STORAGE_VERSION, STORAGE_KEY, minor_version=STORAGE_VERSION_MINOR
         )
-        self._prefs = None
-        self._listeners = []
+        self._prefs: dict[str, Any]
```

Comment:
```suggestion
        self._prefs: dict[str, Any] = {}
```

Reply:
There's no reason to set this in `__init__` and then again in `async_initialize` which must be called before the `CloudPreferences` can be used.
Changed to a class level type annotation instead.